### PR TITLE
Fix : file names to snake case

### LINKED ISSUE #267: File names are inconsistent
config folder contains file name in camel case convention
loaders contains file name in snake case.

we should formalize a style and then format / rename everything accordingly.

### PR DESCRIPTION
**Pull request by Dev-Khant**
## Description

Changes are made to file names, converted to snake case. Also changed all the relevant files where imports were made. In issue it's mentioned to also change `/docs` but I couldn't find any place where changes were necessary. Do let me know if there changes are to be made.

Fixes #267

## Type of change

Please delete options that are not relevant.

- [x] Refactor (does not change functionality, e.g. code style improvements, linting)

## How Has This Been Tested?

I ran `poetry run pytest` for all tests.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [x] Made sure Checks passed

### CODE REVIEW

File: embedchain/config/apps/base_app_config.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import logging
 
-from embedchain.config.BaseConfig import BaseConfig
-from embedchain.config.vectordbs import ElasticsearchDBConfig
+from embedchain.config.base_config import BaseConfig
+from embedchain.config.vectordbs import elasticsearch_db_config
```
Comment (cachho) [inline embedchain/config/apps/base_app_config.py:4]: maybe `vector_db` if we want to nitpick

Comment (cachho) [reply embedchain/config/apps/base_app_config.py:4]: cause we also use `elasticsearch_db`

Comment (Dev-Khant) [reply embedchain/config/apps/base_app_config.py]: I also changed the folder names `vectordb` to `vector_db` and `vectordbs` to `vector_dbs`

**Comment by taranjeet** [comment]
@Dev-Khant : the tests are still failing. let me know in the comments when you are done fixing the errors. Will review and merge.
Great work with the entire PR btw, Its a big and makes the code cleaner.

**Comment by Dev-Khant** [comment]
@taranjeet It's passing all the tests locally. So what am I missing here or where you think the issue might be?

image

**Comment by taranjeet** [comment]
@Dev-Khant : can you push a dummy commit? will try running the tests again

**Comment by Dev-Khant** [comment]
@taranjeet Pushed a dummy commit. Can you please check it again.

**Comment by Dev-Khant** [comment]
@taranjeet @cachho All test have passed, please do review.

**Review by cachho — APPROVED** [review]
looks good to me

**Comment by Dev-Khant** [comment]
@taranjeet Is there any changes needed here?

**Comment by taranjeet** [comment]
Hey @Dev-Khant , nothing here.
thanks for your contribution.
Its a big PR and touches almost everything, so just taking my time properly before merging.
will review again and merge soon

**Comment by Dev-Khant** [comment]
Alright Thanks!

**Comment by cachho** [comment]
Hey, we're going to have to redo this. I think we might as well close this, you're going to have to start over anyways. #267 is still open, and can be taken up again at any time.

**Comment by Dev-Khant** [comment]
Sure will do it

**Pull request closed without merging**